src/pages/AddListingPage/AddListingPage.tsx

```
@@ -0,0 +1,221 @@
+
+            <Input
+              label="Title"
+              placeholder="e.g., iPhone 13 Pro Max 256GB"
+              {...register("title")}
+              error={errors.title?.message}
+            />
+
+            <div>
+              <label className="text-bodySmall mb-2 block font-medium">
+                Description
+              </label>
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)